### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by yardenshoham**
Fixed a small typo

### CODE REVIEW

File: content/en/docs/ambient/usage/add-workloads/index.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ In most cases, a cluster administrator will deploy the Istio mesh infrastructure
 
 ## Enabling ambient mode for an application in the mesh
```
Comment (ydFu) [inline content/en/docs/ambient/usage/add-workloads/index.md:11]:
```suggestion
## Enabling ambient mode for applications in the mesh
```

Comment (ydFu) [reply content/en/docs/ambient/usage/add-workloads/index.md:11]: Correct the comparison article content

**Review by kfaseela — APPROVED** [review]

**Pull request merged**